app/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+
+    <style name="StyleTextInputLayout" parent="TextAppearance.AppCompat">

```

Comment:
rename StyleTextInputLayout -> text_field_style

Reply:
I'm refactor code because it non actual